torch/linalg/__init__.py

```diff
@@ -204,31 +213,42 @@
 slogdet = _add_docstr(_linalg.linalg_slogdet, r"""
 linalg.slogdet(input, *, out=None) -> (Tensor, Tensor)
 
-Calculates the sign and natural logarithm of the absolute value of a square matrix's determinant,
-or of the absolute values of the determinants of a batch of square matrices :attr:`input`.
-The determinant can be computed with ``sign * exp(logabsdet)``.
+Computes the sign and logarithm of the norm of the determinant of a square matrix or batch of square matrices.
 
-Supports input of float, double, cfloat and cdouble datatypes.
+For a complex :attr:`input`, as the determinant is complex in this case, it returns the angle
+and the logarithm of the modulus of the determinant.
 
-.. note:: When given inputs on a CUDA device, this function synchronizes that device with the CPU.
+The returend value decomposition is represented as a namedtuple `(sign, logabsdet)`.
```

Comment:
"returend" -> "returned"

Reply:
"is represented as a namedtuple" -> "is a named tuple"

Reply:
This is used throughout the docs. I used "namedtuple" because it is what was used before. Should I change it to "named tuple"?